README.md

```diff
@@ -1,8 +1,24 @@
 # Content Store
 
-The Content Store is a MongoDB database of almost all published content on GOV.UK.
+The Content Store is a database of almost all published content on GOV.UK.
 See the [Content Store API](./docs/content-store-api.md) for basic usage.
 
+## Deployment in current transitional state
+
+For most of 2023, Publishing Platform team have been working on migrating Content Store from MongoDB to PostgreSQL on Amazon's RDS service. This work is nearing completion, but at the moment there is a non-standard setup for deploying any changes.
+
+* The `main` branch is currently _not in use by any deployed application_
+* The `content-store` and `draft-content-store` applications in all environments are using a container name of `content-store`, but that must be manually built from the `port-to-postgresql` branch
+
+If you need to deploy Content Store, make sure that you:
+* base any pull requests off the `port-to-postgresql` branch (see PR #1085)
+* merge any pull requests into the `port-to-postgresql` branch
+* deploy by running the [Deploy workflow](https://github.com/alphagov/content-store/actions/workflows/deploy.yml)
```

Comment:
Unless you're planning to switch deployment promotion on, it would be helpful to mention that we mean running the workflow for each environment here (because I think promotion is the norm in other apps?)

Reply:
Good call, thanks Mike - done